packages/core/src/middleware/koa-slonik-error-handler.ts

```diff
@@ -21,6 +21,7 @@
 
 import { SchemaLike } from '@logto/schemas';
 import { Middleware } from 'koa';
+import { DatabaseError } from 'pg';
```

Comment:
can we just import type? it'll make sense if we don't have `pg` in our deps

Reply:
after re-consideration i think we should not show db error in API? @simeng-li @wangsijie any ideas?

Reply:
`DatabaseError` is a `class` not type. We cannot just import its type to deal with `DatabaseError`.

Reply:
It's OK to show db error in development mode, and only show 500 without details in production?